checks/http.go

```diff
@@ -43,7 +48,33 @@ func init() {
 	prometheus.MustRegister(responseStatus, sslExpiration)
 }
 
-type HttpChecker struct{}
+type HttpChecker struct {
+	k8s           *kommons.Client
+	specNamespace string
+}
+
+func NewHttpChecker() *HttpChecker {
+	nc := &HttpChecker{}
+
+	kommons, err := pkg.NewKommonsClient()
```

Comment:
We don't want todo this at startup, only when an actual http check is run, so this probably needs to be on HttpCheck rather than HttpChecker